gui/src/installer/view.rs

```diff
@@ -839,6 +840,21 @@ pub fn define_bitcoin<'a>(
     selected_auth_type: &RpcAuthType,
     is_running: Option<&Result<(), Error>>,
 ) -> Element<'a, Message> {
+    let is_loopback = if let RpcAuthType::UserPass = selected_auth_type {
+        if let Some((ip, _port)) = address.value.clone().split_once(':') {
+            let (ipv4, ipv6) = (Ipv4Addr::from_str(ip), Ipv6Addr::from_str(ip));
+            match (ipv4, ipv6) {
+                (_, Ok(ip)) => ip.is_loopback(),
+                (Ok(ip), _) => ip.is_loopback(),
+                _ => false,
+            }
+        } else {
+            false
+        }
+    } else {
+        true
```

Comment:
Is it worth applying the IP check also here in case a remote cookie path has been entered somehow?

Reply:
+1, we might as well show the warning in both cases.

Reply:
done